!Questie/QuestieOptionsForm.lua

```diff
@@ -121,7 +121,9 @@ function Questie:OptionsForm_ApplyOptions()
 
     QuestieConfig.clusterQuests = Questie:toboolean(QO_clusterquests:GetChecked())
 
-    DEFAULT_CHAT_FRAME:AddMessage("Questie Options Applied", 1, 0.75, 0);
+    DEFAULT_CHAT_FRAME:AddMessage("Questie Options Applied", 1, 0.75, 0)
+
+    Questie:Toggle()
```

Comment:
It needs it twice to basically re-draw all the questie stuff. It's the same as pressing the Toggle Questie button twice to turn it off and on to apply all the settings. Most of the other slash commands do the double toggle.